guidelines/content-guidelines/01-content-strategy-k2.md

```diff
@@ -0,0 +1,193 @@
+> Conceptual information might explain the nature and components of a product and describe how it fits into a category of products. Conceptual information helps readers to map their knowledge and understanding to the tasks they need to perform and to provide other essential information about a product, process, or system.
+
+You can use the [concept topic template](../templates/resources/concept.md).
+
+### Task (tutorial) topics
+
+Provide "how-to" instructions that enable users to accomplish a task. Each task topic should tell how to perform a single, specific procedure.
+
+Select a title that describes the task that's accomplished, not the documented software feature - for example, "Define ressource consumption", not "Select a profile". You can use gerund form "Selecting...", imperative "Select...", or "How to select...".
+
+With regards to structure, it’s nice to have an **introductory paragraph** ("why would I want to do this task?"), **prerequisites** if needed, then the **steps**, and finally the expected **result** that shows the operation was successful.
+It's good practice to have 5-9 steps; anything longer can probably be split.
+
+*For longer code blocks, I'd like to use an expandable section as described [here](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab) - but not sure whether our website supports this.*
+
+> [- Dita Task Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-task-topic.html)
+> A task information type answers the "How do I?" question by providing precise step-by-step instructions detailing the requirements that must be fulfilled, the actions that must be performed, and the order in which the actions must be performed. The task topic includes sections for describing the context, prerequisites, expected results, and other aspects of a task.
+
+You can use the [task topic template](../templates/resources/task.md).
+
+### Reference topics
+
+Typically organized into one or more sections containing a list or table with data that is typically “looked up” rather than memorized.
+
+Use nominal style for the title, for example, "Configuration Parameters".
+
+*In our case, architecture diagrams could fall into this category – we could also choose to define them as a separate type – TBD.*
+
+> [- DITA Reference Topics](http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-reference-topic.html)
+> Reference topics provide quick access to fact-based information. In technical information, reference topics are used to list product specifications and parameters, provide essential data, and provide detailed information on subjects such as the commands in a programming language.
+
+### Troubleshooting topics
+
+Provide a condition that the reader may want to correct, followed by one or more descriptions of its cause and suggested remedies.
```

Comment:
```suggestion
Provide a condition that the reader may want to correct, followed by descriptions of one or more of its cause and suggested remedies.
```

Reply:
I think this has a slightly different meaning?